bin/delayed_job

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/home/bottlenose/.rbenv/shims/ruby
```

Comment:
True, but `/usr/bin/env ruby` didn't run the correct program.  Anyway this is moot after your PR, right?